### PR TITLE
Poster Pack 2 - Beyond Posters

### PR DESCRIPTION
**Pull request by EOBGames**
## About The Pull Request
Adds a new selection of posters, poster-like objects, and wall decorations.

First up, the new posters:
_Contraband Posters_
image
_Official Posters_
image

New billboards:
image
cvr_billboard

New entirely with this PR, shipping containers! These sprites were adapted from a base from TGMC, with brands and designs more fitting to our universe here on TG:
_Station-Legal_
image
_Syndicate_
image

Another new addition, flags! These draw upon Common Core lore, representing several major factions from the lorebase as a new mapping tool:
image

Finally, there are two interactive wall objects- wall clocks, and calendars:
image

The wall clock shows both the "real time", represented as the Earth time, or Treaty Coordinated Standard Time, and the station time, represented as the local time, or Coalition Standard Time:
image

The calendar gives the in-universe date, as well as any active holidays:
image

## Why It's Good For The Game
This adds a lot of fun aesthetic additions that I think will benefit our mapping team in creating new spaces and breathing new life into older ones. Additionally, it helps make our Common Core lore a bit more tangible to the playerbase, bringing it away from being "just on a wiki" and into the game proper.

## Changelog

:cl:
add: A whole host of new posters have just become available.
add: For the architects among you: a number of new aesthetic options are available for mapping with. Let your creativity free!
/:cl:

### CODE REVIEW

File: code/game/objects/structures/signs/signs_interactive.dm [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	desc = "It's an old-school wall calendar. Sure, it might be obsolete with modern technology, but it's still hard to imagine an office without one."
+	icon_state = "calendar"
+
+MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/calendar, 32)
+
+/obj/structure/sign/calendar/examine(mob/user)
+	. = ..()
+	. += span_info("The current date is: [time2text(world.realtime, "DDD, MMM DD")], [GLOB.year_integer+540].")
+	if(SSevents.holidays)
+		. += span_info("Events today:")
```
Comment (LemonInTheDark) [inline code/game/objects/structures/signs/signs_interactive.dm:24]: is today best here? sometimes we get things with week in the name and such
maybe just Holidays: ?

File: code/game/objects/structures/signs/signs_interactive.dm [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		for(var/holidayname in SSevents.holidays)
+			var/datum/holiday/holiday = SSevents.holidays[holidayname]
+			. += span_info("[holiday.name]")
```
Comment (LemonInTheDark) [inline code/game/objects/structures/signs/signs_interactive.dm:27]: is holidayname not already what you want?

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/signs/signs_interactive.dm (hunk 1 above)
Comment (EOBGames) [reply code/game/objects/structures/signs/signs_interactive.dm:24]: I had considered that, but figured in the end that an event occurring across multiple days still occurs individually on each of those days. That said, shortening it to just "Events" might be smarter, yeah.

File: code/game/objects/structures/signs/signs_interactive.dm (hunk 2 above)
Comment (Jackal-boop) [reply code/game/objects/structures/signs/signs_interactive.dm:27]: id love to see a nakimura poster

**Comment by Jackal-boop** [comment]
id love to see a nakimura poster

**Review by Twaticus — APPROVED** [review]

**Pull request merged**